scripts/validateCode

```diff
@@ -4,6 +4,11 @@
 
 . "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/build"
 
-build validateCode
+build scalafmtCheckAll scalafmtSbtCheck || (
+    echo "[error] ERROR: Scalafmt test failed."
+    echo "[error] To fix, format your sources using 'sbt scalafmtAll scalafmtSbt' before submitting a pull request."
+    echo "[error] Additionally, please squash your commits (eg, use git commit --amend) if you're going to update this pull request."
+    false
+  )
```

Comment:
Hmmm, should we get rid of the `validateCode` alias in `build.sbt` then?

Reply:
Hmm I think we have two choices here, either change it here to use the command alias like `build validateCode || ...` or keep it like that, but then we also have to update the README.md because there is a reference to run `sbt validateCode` before pushing code. I would vote for the command alias as it wraps both scalafmt tasks, what do you think?

Reply:
I'm fine either way. As long as we keep things consistent :-)

Keeping `sbt validateCode` probably requires fewer changes.